Docs: review third-party scripts (#2155)
vtex/faststore: apps/site/pages/docs/handling-third-party-scripts.mdx

<header>

# Adding third-party scripts

</header>

In this guide, learn how to add third-party scripts to FastStore projects.

Third-party scripts are elements embedded in a webpage but served from a different domain than the one the user visits. 
Examples of these scripts include social media buttons, video players, ads, analytics tools, and other elements contributing to a more dynamic and interactive web experience.

## Before you begin

<Steps>

### Choose essential third-party scripts
To maintain your website's performance without compromising the functionality offered 
by third-party scripts, consider incorporating only those third-party scripts that
enhance the value of your website.

### Update the `@faststore/core` version
Ensure your project uses `@faststore/core` version `2.1.82` or above. If not, follow these steps:

> To do this, do the following:

Author: ```suggestion
```

1. Open the FastStore code project.
2. Navigate to the `package.json` file.
3. Find the `@faststore/core` package and change its version to `2.1.82`. 
4. In your terminal, run `yarn install` to update the package in the project.

### Verify compatibility with Partytown

> FastStore uses [Partytown](https://partytown.builder.io/) for script execution, and not all third-party scripts may be compatible with Partytown. 
> Test third-party scripts with Partytown before implementing them. 
> If any issues occur, avoid using those particular scripts. 
> Refer to the official Partytown documentation on [Common Services](https://partytown.builder.io/common-services) for compatibility details.

Author: ```suggestion
FastStore uses [Partytown](https://partytown.builder.io/) for script execution. Not all third-party scripts may be compatible with Partytown, so we strongly recommend testing third-party scripts with Partytown before implementing them. If any issues occur, avoid using those particular scripts. 
```
